contractcourt/anchor_resolver.go

```diff
@@ -176,13 +176,13 @@ var _ ContractResolver = (*anchorResolver)(nil)
 
 // Launch offers the anchor output to the sweeper.
 func (c *anchorResolver) Launch() error {
-	if c.launched {
+	if c.isLaunched() {
 		c.log.Tracef("already launched")
```

Comment:
Is this concurrent case due to the tests or is there also the possibility in the normal program flow that the Launch is called concurrently ?

Reply:
yeah it's possible multiple places call this.